attrd/commands.c

```diff
@@ -294,6 +311,54 @@ attrd_client_update(xmlNode *xml)
 
 /*!
  * \internal
+ * \brief Respond to client clear-failure request
+ *
+ * \param[in] xml         Request XML
+ */
+void
+attrd_client_clear_failure(xmlNode *xml)
+{
+#if 0
+    /* @TODO This would be most efficient, but there is currently no way to
+     * verify that all peers support the op. If that ever changes, we could
+     * enable this code.
+     */
+    if (all_peers_support_clear_failure) {
```

Comment:
There are enough messages sent around that you'd quickly get one from each, track it and check everyone is >= 2?